test/integration/targets/sqitch/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+- block:
+  - name: Create dir for resources
+    file: path=/tmp/sqitch_test state=directory mode=0755
+  
+  - name: Copy resources to tmp dir
+    copy: src=files/ dest=/tmp/sqitch_test
+  
+  - name: Create DB for tests
+    postgresql_db: name=anstest encoding='UTF-8' state=present template=template0
+  
+  - name: Install sqitch
+    package: name=sqitch state=present
+    when: ansible_os_family == "Debian"
+  
+  - name: Install CPAN
+    package: name={{ item }} state=present
+    with_items:
+      - perl-devel
+      - perl-CPAN
+      - perl-DBD-Pg
+    when: ansible_os_family == "RedHat"
+  
+  - name: Install cpanminus
+    shell: "curl -L http://cpanmin.us | perl - --sudo App::cpanminus"
```

Comment:
Out of curiosity, why aren't you using the `cpanm` Ansible module for this?

Reply:
To use the `cpanm` module I need to install cpanm first and this is the one of recommended ways to do it - https://metacpan.org/pod/App::cpanminus#Installing-to-system-perl

Reply:
Ah, TIL 👍